Makefile.am

```diff
@@ -58,6 +58,7 @@ shellcheck:
 			scripts/zfs.sh \
 			scripts/commitcheck.sh \
 			$$(find cmd/zed/zed.d/*.sh -type f); \
+			$$(find cmd/zpool/zpool.d/* -executable); \
```

Comment:
You need to remove the semicolon from the previous line. Your scripts aren't being passed to shellcheck atm.